v3.20/backports/py3-rosdistro/force-to-enable-alpine-platform.patch

```diff
@@ -1,23 +1,13 @@
 diff --git a/src/rosdistro/distribution_file.py b/src/rosdistro/distribution_file.py
-index c330329..d31fcd6 100644
+index c330329..710978f 100644
 --- a/src/rosdistro/distribution_file.py
 +++ b/src/rosdistro/distribution_file.py
-@@ -74,6 +74,19 @@ class DistributionFile(object):
+@@ -74,6 +74,9 @@ class DistributionFile(object):
                      assert os_code_name not in self.release_platforms[os_name], "Distribution '%s' specifies the os_code_name '%s' multiple times for the os_name '%s'" % (self.name, os_code_name, os_name)
                      self.release_platforms[os_name].append(os_code_name)
  
 +        # Tweak for alpine platform
-+        self.release_platforms['alpine'] = []
-+        if name == 'kinetic':
-+            self.release_platforms['alpine'].append('3.7')
-+        if name == 'melodic':
-+            self.release_platforms['alpine'].append('3.8')
-+        if name == 'noetic':
-+            self.release_platforms['alpine'].append('3.11')
-+            self.release_platforms['alpine'].append('3.14')
-+            self.release_platforms['alpine'].append('3.17')
-+        if name == 'humble':
-+            self.release_platforms['alpine'].append('3.17')
++        self.release_platforms['alpine'] = ['3.20']
```

Comment:
As rosdep can be used to resolve dep keys for other OS version using `--os=` flag, it would be better to keep the other versions